### ISSUE TITLE
Excel dates 

### ISSUE THREAD

**Issue by calvinmetcalf** [Quartz/bad-data-guide#2]
Arn't they actually December 31st 1899 or something similar slightly off. Due to leap year bug or something.

**Comment by hydrosquall** [comment]
@calvinmetcalf could you post an example of the issue you're describing (is it [this](https://support.microsoft.com/en-us/kb/214326) one?)

**Comment by bdcaf** [comment]
Yes the dates are stored as number counting from "1899-12-30" (is day 0).  
Seems the didn't consider that if year is divisible by 100 it is not a leap year.

Fun fact - there is a second date system used by excel for mac which starts on "1904-01-01"
(https://support.microsoft.com/en-us/help/214330/differences-between-the-1900-and-the-1904-date-system-in-excel - interesting they don't mention the leap year bug there...)
